packages/core/src/httpClient/retryValidator.ts

```diff
@@ -50,6 +50,10 @@ export const createUniversalRetryValidator = (
   if (retryCount >= maxRetries) {
     return false;
   }
+  // Always retry http 429 too many requests
+  if (response.status === 429) {
+    return true;
```

Comment:
Retry is nice. However, are we respecting `Retry-After` headers here?
Otherwise I think this might get stuck in a retry loop

Reply:
Not any headers I think, but the maxRetries will hit before this.